packages/react-components/src/components/Datepicker/DatePicker.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import ReactDayPicker from 'react-day-picker';
+import cx from 'classnames';
+import DatePickerNavbar from './DatePickerNavbar';
+import { IDatePickerProps } from './types';
+import { isDateWithinRange } from './helpers';
+import styles from './DatePicker.module.scss';
+
+const baseClass = 'date-picker';
+
+const defaultDayRenderer = (day: Date): JSX.Element => {
+  const date = day.getDate();
+
+  return (
+    <div className={styles[`${baseClass}__day-wrapper`]}>
+      <div className={styles[`${baseClass}__day-content`]}>{date}</div>
+    </div>
+  );
+};
+
+const DatePickerComponent: React.FC<IDatePickerProps> = (props) => {
+  const {
+    classNames,
+    range,
+    month: propsMonth,
+    toMonth,
+    fromMonth,
+    firstDayOfWeek: propsFirstDayOfWeek,
+    numberOfMonths,
+    navbarElement,
+    renderDay,
+    innerRef,
+    ...restProps
+  } = props;
+
+  const [month, setMonth] = React.useState(propsMonth || new Date());
+
+  React.useEffect(() => {
```

Comment:
do we need to supply `React.` namespace everywhere? you could import it directly

Reply:
We can drop importing `* as React` if whenever possible